.flake8

```diff
@@ -43,6 +43,9 @@ per-file-ignores =
     dpnp/linalg/__init__.py: F401
     dpnp/random/__init__.py: F401
     dpnp/dpnp_iface.py: D205
```

Comment:
I think it would be better to ignore this rule in all `dpnp` files. 
Cause the line break does not affect the function description in the generated documentation.

Reply:
Totally agree, I'll disable the warning globally